cfme/common/vm.py

```diff
@@ -178,6 +178,12 @@ def ip_address(self):
         """Fetches IP Address of VM"""
         return self.mgmt.ip
 
+    @property
+    def mac_address(self):
+        """Fetches MAC Address of VM"""
+        view = navigate_to(self, "Details", use_resetter=False)
+        return view.entities.summary('Properties').get_text_of('MAC Address')
```

Comment:
**Question** Why could we not use wrapanapi to pull the mac address? Or do you want to get it from CFME exclusively?

Reply:
Will resolve this as TODO as per the discussion we had offline.